pylot/perception/detection/detection_operator.py

```diff
@@ -206,10 +207,10 @@ def __run_model(self, image_np):
         infer = self._model.signatures['serving_default']
         result = infer(tf.convert_to_tensor(value=image_np_expanded))
 
-        boxes = result['boxes']
-        scores = result['scores']
-        classes = result['classes']
-        num_detections = result['detections']
+        boxes = result[self._model_outputs['boxes']]
+        scores = result[self._model_outputs['scores']]
+        classes = result[self._model_outputs['classes']]
+        num_detections = result[self._model_outputs['detections']]
```

Comment:
Instead of using hard-coded keys into the `result` dictionary, we use keys from the `*.outputs` file specified by the new config flag.